tcf_website/tests/test_utils.py

```diff
@@ -30,38 +30,42 @@ def setup(obj):
         season='FALL',
         number=1108
     )
+    obj.incomplete_semester = Semester.objects.create(
+        year=2019,
+        number=1198)
```

Comment:
I think this should be either
```suggestion
    obj.incomplete_semester = Semester.objects.create(year=2019, number=1198)
```
or
```suggestion
    obj.incomplete_semester = Semester.objects.create(
        year=2019,
        number=1198,
    )
```